ci-jobs/templates/bundle-template.yml

```diff
@@ -2,7 +2,7 @@
 jobs:
   - job: bundle_and_upload_appium_zip
     pool:
-      vmImage: 'macOS 10.14'
+      vmImage: 'macOS 11'
```

Comment:
I believe it's macOS-11 (with a dash)
We could also probably just use latest as this script does not have hard dependencies on image component versions?